ols/app/models/config.py

```diff
@@ -271,27 +271,62 @@ def validate(self) -> None:
         pass
 
 
-class LoggerConfig(BaseModel):
-    """Logger configuration."""
+class LoggingConfig(BaseModel):
+    """Logging configuration."""
 
-    default_level: Optional[int | str] = None
+    app_log_level: Optional[str] = None
+    httpcore_log_level: Optional[str] = None
+    httpx_log_level: Optional[str] = None
+    urllib3_log_level: Optional[str] = None
 
     def __init__(self, data: Optional[dict] = None):
         """Initialize configuration and perform basic validation."""
         super().__init__()
         if data is None:
             return
-        level = logging.getLevelName(data.get("default_level", "INFO"))
-        if level is None:
+        app_log_level = logging.getLevelName(data.get("app_log_level", "info").upper())
```

Comment:
when it will be None?

Reply:
Maybe some value for `getLevelName` exists that would return `None`. But most likely, it will be never `None`. I didn't want to change the actual validation here. Seems there is still some room for improvement. But I can, if you think it belongs here.

Reply:
Yea, this would fail when you provide `app_log_level: false`.

Reply:
refactored